### ISSUE TITLE
Templates get over-ridden

### ISSUE THREAD

**Issue by jmcd040** [writemonkey/wm3#257]
WM3 Beta8
I've been developing my own css templates starting with one of the templates that came with WM3. However it seems that some of my css is over-ridden. For example I've tried setting the color and style of links without being able to affect the way they appear in the preview and html. When I inspect the html using Chrome, I find that the html file you produce has the css template at the top, then has a 1<style> tag that over-rides some of the css formatting. For example I try to set the `a` tag in css to be colored and underlined, but that styling is never seen. Am I right about the fact that you over-ride some of the css styling?

**Comment by writemonkey** [comment]
You are right.
i.

**Comment by jmcd040** [comment]
Is there a way to get around this?

**Comment by writemonkey** [comment]
Yes, you can open `plugins/preview/preview.js` and find parts of the code with css in it. You can edit or remove those css rules. There may be unforeseen problems doing that but if you are careful enough ... 
i.

**Comment by jmcd040** [comment]
Ok, in preview.js I removed `!important` from the `a` element and change line 444 to reverse the order of loading wmColorsCSS and templateCSS so wmColorsCSS is loaded first allowing the template to have the last say. The new line is this:
`return '<!DOCTYPE html><html>\n<head>\n<meta http-equiv="Content-Type" content="text/html; charset=utf-8">\n<script>\n' + templateJS + '\n</script>\n' + wmColorsCSS + '<style type="text/css">\n' + templateCSS + '</style>\n'  + num + '\n<title>' + (monkey._currentDoc && monkey._currentDoc.nam ? monkey._currentDoc.nam.substring(0, 20) : 'WM export') + '</title>\n   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.css" integrity="sha384-9eLZqc9ds8eNjO3TmqPeYcDj8n+Qfa4nuSiGYa6DjLNcv9BtN69ZIulL9+8CqC9Y" crossorigin="anonymous"><script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.js" integrity="sha384-K3vbOmF2BtaVai+Qk37uypf7VrgBubhQreNQe9aGsz9lB63dIFiQVlJbr92dw2Lx" crossorigin="anonymous"></script><script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/contrib/auto-render.min.js" integrity="sha384-kmZOZB5ObwgQnS/DuDg6TScgOiWWBiVt0plIRkZCmE6rDZGrEOQeHM5PcHi+nyqe" crossorigin="anonymous" onload=\'renderMathInElement(document.body, {throwOnError: false, strict: false, displayMode: false, delimiters: [{left: "$$$", right: "$$$", display: true}, {left: "$$", right: "$$", display: false}]});\'></script>\n </head>\n<body style="font-size:.8em;">\n<div id="wrapper">\n' + content + '\n</div></body>\n</html>';`
**This worked - (so far).**
